src/main/java/teammates/ui/webapi/FeedbackSessionClosingRemindersAction.java

```diff
@@ -54,11 +56,14 @@ public JsonResult execute() {
                 continue;
             }
 
-            List<EmailWrapper> emailsToBeSent =
-                    emailGenerator.generateFeedbackSessionClosingWithExtensionEmails(feedbackSession, deadlineExtensions);
+            List<DeadlineExtensionAttributes> filteredDeadlineExtensions =
+                    filterValidDeadlineExtensions(deadlineExtensions, feedbackSession);
```

Comment:
Just wondering: is `filteredDeadlineExtensions` or `validDeadlineExtensions` more appropriate?

Reply:
I think I prefer valid, filtered is slightly more ambiguous (filter out what?). I'll make the change!